### PR TITLE
Home page

### PR DESCRIPTION
**Pull request by jlhogan**
Still missing the guided search entry point. will leave that til we actually implement guided search.

### CODE REVIEW

**Review by cathydeng — APPROVED** [review]
couple bits of enhancement (not blockers):
- the space between the image and the grey line at the nav is kind of awk @ width >1024. i think this is from the usa-section class on the div below the header in Layout.tsx?
- for the content cards, would be nice for the images at the top to take up the same amount of vertical space, so that the card titles are aligned on desktop and look nice next to each other
- would be nice to wrap all phone numbers in a span w/ whitespace nowrap to ensure that they don't break across lines on some screen sizes

File: src/translations/en/pages.json [hunk 1]
```diff
@@ -5,7 +5,22 @@
   },
   "home": {
     "heading": "Find behavioral health care that's right for you.",
-    "zipPrompt": "Enter your zip code to find care near your area"
+    "zipPrompt": "Enter your zip code to find care near your area",
+    "_coloradoCrisisService": {
+      "heading": "If you’re experiencing crisis",
+      "content": "No matter what you’re going through, help is available. Call Colorado Crisis Services at 844-493-TALK (8255) or text TALK to 38255.",
+      "cta": "Learn more at ColoradoCrisisServices.org"
```
Comment (cathydeng) [inline src/translations/en/pages.json:12]: it seems kinda weird for the CTA to have a url. seems like "Learn more" would be sufficient

File: src/translations/en/pages.json [hunk 2]
```diff
@@ -5,7 +5,22 @@
...
+    },
+    "_iMatter": {
+      "heading": "Free therapy for youth",
+      "content": "I Matter can connect youth 18 years of age and younger (21 years or younger if receiving special education services) with a therapist for up to 6 free and confidential counseling sessions.",
+      "cta": "Visit IMatterColorado.org To find a therapist and schedule an appointment."
```
Comment (cathydeng) [inline src/translations/en/pages.json:17]: same as above. "Find a therapist" or "Find a therapist and schedule an appointment" seem like better CTAs

File: src/translations/en/pages.json (hunk 1 above)
Comment (jlhogan) [reply src/translations/en/pages.json:12]: I'll let @admoorgit weigh in on content

**Review by cathydeng — APPROVED** [review]
🏠

**Pull request merged**